### PR TITLE
fix component options not working when passed false as parameter

### PR DESCRIPTION
**Pull request by XcrossD**
There seems to have line ending issues, but the changes are basically in the constructor set state.

### CODE REVIEW

**Comment by XcrossD** [comment]
I'm also working on adding dot indices, is there a style guideline that you follow so that I can style the dots accordingly?
I have something like this in mind:
example

File: build/index.js [hunk 1]
```diff
@@ -331,12 +331,12 @@ exports.default = Slideshow;
 /* 4 */
 /***/ (function(module, exports, __webpack_require__) {
 
-exports = module.exports = __webpack_require__(5)(undefined);
+exports = module.exports = __webpack_require__(5)(false);
```
Comment (Pau1fitz) [inline build/index.js]: thanks for these changes. Just wondering why have you passed in `false` here, as opposed to `undefined`?

**Comment by Pau1fitz** [comment]
regarding the dots, there is no style guide. I just used css for the styling here, feel free to add the dots! :-D

File: build/index.js (hunk 1 above)
Comment (XcrossD) [reply build/index.js]: I only changed the javascript source file and ran "npm run build", so I'm not sure why it is different.

**Comment by XcrossD** [comment]
Okay I have finished adding the dots so another commit incoming. Also I noticed that for the "effects" option, using a string instead of a boolean for the default value is probably better, so maybe you can do that? Saves us both some time.

**Comment by XcrossD** [comment]
The line ending problem still persists, so the main changes are the following:
index.js
Slideshow.css -> the part around .show-index

**Comment by mad-briller** [comment]
could use this being merged soon, thank you !

**Pull request merged**